### PR TITLE
Update GET top N api documentation

### PR DESCRIPTION
**Pull request by dzane17**
### Description
Clarify GET top N api documentation

### Issues Resolved
Resolves https://github.com/opensearch-project/query-insights/issues/80 & https://github.com/opensearch-project/query-insights/issues/83

### Version
2.15+

### Frontend features
_If you're submitting documentation for an OpenSearch Dashboards feature, add a video that shows how a user will interact with the UI step by step. A voiceover is optional._ 

### Checklist
- [x] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by dzane17** [comment]
@kolchfa-aws Ready for doc review. Thanks!

File: _observing-your-data/query-insights/top-n-queries.md [hunk 1]
```diff
@@ -96,6 +94,9 @@ GET /_insights/top_queries?type=memory
 ```
 {% include copy-curl.html %}
 
+If you are not seeing results, verify top N is enabled for the target metric type and ensure search requests were sent during the current time window.
```
Comment (kolchfa-aws) [inline _observing-your-data/query-insights/top-n-queries.md:97]: @dzane17 Here's a rewording suggestion. Could you also clarify what "current" means in this context?
```suggestion
If your query returns no results, ensure that top N is enabled for the target metric type and that search requests were made within the current time window.
```

**Review by kolchfa-aws — COMMENTED** [review]
Thank you, @dzane17! Could you give a clarification to the "current time window"? Otherwise, looks good.

**Review by ansjcy — APPROVED** [review]

**Comment by dzane17** [comment]
@kolchfa-aws I've added your suggestion and included a link to "configuring window size" to clarify.

**Review by kolchfa-aws — APPROVED** [review]
Thank you, @dzane17! LGTM. Applying "Done but waiting to merge" label to put this in the editorial queue starting 9/11.

File: _observing-your-data/query-insights/top-n-queries.md [hunk 2]
```diff
@@ -96,6 +94,9 @@ GET /_insights/top_queries?type=memory
 ```
 {% include copy-curl.html %}
 
+If your query returns no results, ensure that top N is enabled for the target metric type and that search requests were made within the current [time window](#configuring-the-window-size).
```
Comment (deshsidd) [inline _observing-your-data/query-insights/top-n-queries.md:97]: Do we want to mention that the sorting of the results will be based on the metric type provided?

File: _observing-your-data/query-insights/top-n-queries.md
Comment (deshsidd) [inline _observing-your-data/query-insights/top-n-queries.md:1]: Monitoring the top N queries in query insights features can help you gain real-time insights into the top queries with high latency within a certain time frame (for example, the last hour).

need to included cpu and memory too here

Comment (ansjcy) [reply _observing-your-data/query-insights/top-n-queries.md:1]: +1, let's update this in this PR.

**Comment by dzane17** [comment]
@deshsidd @ansjcy Updated intro and added sentence about sorting by metric type

**Review by ansjcy — APPROVED** [review]

**Comment by ansjcy** [comment]
Thanks @dzane17 , the change looks good to me.

**Review by deshsidd — APPROVED** [review]

File: _observing-your-data/query-insights/top-n-queries.md [hunk 3]
```diff
@@ -7,7 +7,7 @@ nav_order: 10
 
 # Top N queries
 
-Monitoring the top N queries in query insights features can help you gain real-time insights into the top queries with high latency within a certain time frame (for example, the last hour). 
+Monitoring the top N queries using query insights allows you to gain real-time visibility into the queries with the greatest latency or resource consumption over a specified time period (for example, the last hour).
```
Comment (natebower) [inline _observing-your-data/query-insights/top-n-queries.md:10]:
```suggestion
Monitoring the top N queries using query insights allows you to gain real-time visibility into the queries with the highest latency or resource consumption in a specified time period (for example, the last hour).
```

File: _observing-your-data/query-insights/top-n-queries.md [hunk 4]
```diff
@@ -96,6 +96,9 @@ GET /_insights/top_queries?type=memory
 ```
 {% include copy-curl.html %}
 
+If your query returns no results, ensure that top N is enabled for the target metric type and that search requests were made within the current [time window](#configuring-the-window-size).
```
Comment (natebower) [inline _observing-your-data/query-insights/top-n-queries.md:99]: Should "top N" be in code font or otherwise specially formatted?

**Review by natebower — COMMENTED** [review]
@dzane17 @kolchfa-aws Please see my comments and changes and let me know if you have any questions. Thanks!

File: _observing-your-data/query-insights/top-n-queries.md (hunk 4 above)
Comment (kolchfa-aws) [inline _observing-your-data/query-insights/top-n-queries.md:99]:
```suggestion
If your query returns no results, ensure that top N query monitoring is enabled for the target metric type and that search requests were made within the current [time window](#configuring-the-window-size).
```

Comment (kolchfa-aws) [reply _observing-your-data/query-insights/top-n-queries.md:99]: No. But I clarified so it reads better.

**Pull request merged**